L/licensecheck/bundled/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import "C"
+import (
+    "github.com/google/licensecheck"
+
+    "unsafe"
+)
+
+//export License
+func License(msg *C.char) (**C.char, int, float64) {
+    bytes := []byte(C.GoString(msg))
+    cov := licensecheck.Scan(bytes)
+
+    // https://stackoverflow.com/a/41493208
+    cArray := C.malloc(C.size_t(len(cov.Match)) * C.size_t(unsafe.Sizeof(uintptr(0))))
+    a := (*[1<<30 - 1]*C.char)(cArray)
+
+    for idx, m := range cov.Match {
+        a[idx] = C.CString(m.ID)
+    }
+    
+    return (**C.char)(cArray), len(cov.Match), cov.Percent
+}
+
+func main() {}
```

Comment:
Is this function needed? If it's like C, `main` is needed only for executables

Reply:
Apparently it is; without it, I get a compiler error
```bash
runtime.main_main·f: function main is undeclared in the main package
```
(compiling a shared lib, not an executable)